WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Multi-package/SinglePackageHazmatDeclaration.swift

```diff
@@ -78,47 +79,69 @@ struct SinglePackageHazmatDeclaration: View {
+                static let hazmatInstructionsFirstSection = NSLocalizedString("Potentially hazardous material includes items such as batteries, " +
+                                                                              "dry ice, flammable liquids, aerosols, ammunition, fireworks, nail " +
+                                                                              "polish, perfume, paint, solvents, and more. Hazardous items must " +
+                                                                              "ship in separate packages.",
+                                                                              comment: "Instructions for hazardous package shipping")
+                static let hazmatInstructionsSecondSection = NSLocalizedString("Learn how to securely package, label, and ship HAZMAT through " +
+                                                                               "USPS® at",
+                                                                               comment: "Instructions for hazardous package shipping, the rest of the" +
+                                                                               "phrase is a link that will direct the user to a website")
+                static let hazmatInstructionsSecondSectionLink = NSLocalizedString("www.usps.com/hazmat", comment: "A clickable text link that will" +
+                                                                                   "redirect the user to a website")
+                static let hazmatInstructionsThirdSection = NSLocalizedString("Determine your product's mailability using the",
+                                                                              comment: "Instructions for hazardous package shipping the rest of the" +
+                                                                              "phrase is a link that will direct the user to a website")
+                static let hazmatInstructionsThirdSectionLink = NSLocalizedString("USPS HAZMAT Search Tool", comment: "A clickable text link that will" +
+                                                                                  "redirect the user to a website")
+                static let hazmatInstructionsFourthSection = NSLocalizedString("WooCommerce Shipping does not currently support HAZMAT shipments "
+                                                                               + "through",
+                                                                               comment: "Instructions for hazardous package shipping the rest of the" +
+                                                                               "phrase is a link that will direct the user to a website")
+                static let hazmatInstructionsFourthSectionLink = NSLocalizedString("DHL Express", comment: "A clickable text link that will" +
+                                                                                   "redirect the user to a website")
```

Comment:
nit: are the indentation changes intentional?

Reply:
Oops 😬 not intentional! Fixed in https://github.com/woocommerce/woocommerce-ios/pull/10598/commits/cc05a05c043fe5acbd185ee3c3819e98ac6acb03